closed/GensrcJ9JCL.gmk

```diff
@@ -49,10 +49,14 @@ endif # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
 # OpenJ9 CRIU only supports Linux, so we only need to consider the unix sub-directory.
 OPENJDK_SOURCE_PROCESS_ENVIRONMENT := $(TOPDIR)/src/java.base/unix/classes/java/lang/ProcessEnvironment.java
 OPENJDK_STAGED_PROCESS_ENVIRONMENT := $(patsubst $(TOPDIR)/%,$(SUPPORT_OUTPUTDIR)/overlay/%,$(OPENJDK_SOURCE_PROCESS_ENVIRONMENT))
-
 $(OPENJDK_STAGED_PROCESS_ENVIRONMENT) : $(OPENJDK_SOURCE_PROCESS_ENVIRONMENT)
 	$(call install-file)
 
+OPENJDK_SOURCE_SECURITY := $(TOPDIR)/src/java.base/share/classes/java/security/Security.java
+OPENJDK_STAGED_SECURITY := $(patsubst $(TOPDIR)/%,$(SUPPORT_OUTPUTDIR)/overlay/%,$(OPENJDK_SOURCE_SECURITY))
+$(OPENJDK_STAGED_SECURITY) : $(OPENJDK_SOURCE_SECURITY)
+	$(call install-file)
```

Comment:
The list of 'overlay' files is likely to grow and should be handled in a more extensible way:
```
# OpenJ9 CRIU only supports Linux, so we only need to consider the share and unix sub-directories.

$(eval $(call SetupCopyFiles,COPY_OVERLAY_FILES, \
	SRC := $(TOPDIR), \
	DEST := $(SUPPORT_OUTPUTDIR)/overlay, \
	FILES := \
		src/java.base/share/classes/java/security/Security.java \
		src/java.base/unix/classes/java/lang/ProcessEnvironment.java \
	))
```

Reply:
done